contracts/src/BackedCommunityTokenV1.sol

```diff
@@ -28,12 +28,18 @@ contract BackedCommunityTokenV1 is
         categoryIdToDisplayName[totalCategoryCount++] = displayName;
     }
 
-    function addSpecialAccessory(address artContract)
-        external
-        override
-        onlyOwner
-    {
-        accessoryIdToArtContract[totalSpecialyAccessoryCount++] = artContract;
+    function addSpecialAccessory(
+        IBackedCommunityTokenV1.Accessory memory accessory
+    ) external override onlyOwner {
+        accessoryIdToAccessory[
+            totalSpecialyAccessoryCount++
```

Comment:
this will start at 0 and increment after this call. just want to make sure that is intended

Reply:
yeah thats fine